chore: docs update - unlimited event copy (#2813)
rudderlabs/rudder-server: README.md

- **Privacy and Security**: You can collect and store your customer data without sending everything to a third-party vendor. With RudderStack, you get fine-grained control over what data to forward to which analytical tool.

> - **Unlimited Events**: Event volume-based pricing of most of the commercial systems is broken. With RudderStack Open-Source, you will be able to collect as much data as possible without worrying about overrunning your event budgets.

Reviewer: ```suggestion
- **Unlimited Events**: Event volume-based pricing of most of the commercial systems is broken. With RudderStack Open Source, you can collect as much data as possible without worrying about overrunning your event budgets.
```
